prov-api/routes/model_run/model_run.py

```diff
@@ -2,17 +2,21 @@
 from KeycloakFastAPI.Dependencies import ProtectedRole
 from fastapi import APIRouter, Depends, HTTPException
 from ProvenaInterfaces.ProvenanceAPI import *
+from helpers.registry_helpers import update_model_run_in_registry
 from helpers.workflows import register_and_lodge_provenance
-from helpers.entity_validators import RequestStyle
+from helpers.entity_validators import RequestStyle, validate_model_run_study_linking
 from ProvenaInterfaces.SharedTypes import Status
+from ProvenaInterfaces.RegistryModels import ItemModelRun
 from helpers.validate_model_run_record import validate_model_run_record
 from helpers.auth_helpers import *
-from helpers.job_api_helpers import submit_model_run_lodge_job, submit_batch_lodge_job
-from ProvenaInterfaces.AsyncJobModels import ProvLodgeModelRunPayload, ProvLodgeBatchSubmitPayload
+from helpers.job_api_helpers import submit_model_run_lodge_job, submit_batch_lodge_job, submit_model_run_lodge_only_job
+from ProvenaInterfaces.AsyncJobModels import ProvLodgeModelRunPayload, ProvLodgeBatchSubmitPayload, ProvLodgeModelRunLodgeOnlyPayload
 from config import get_settings, Config
 
 router = APIRouter()
 
+edit_root_prefix = "/edit"
```

Comment:
can you make this CAPS_CASE to indicate it's a constant

Reply:
yes - good point